benchmarking/commons/launch_remote_local.py

```diff
@@ -110,42 +116,121 @@ def launch_remote(
         command line arguments
     :param extra_args: Extra arguments for command line parser, optional
     """
-    args, method_names, seeds = parse_args(methods, extra_args)
-    experiment_tag = args.experiment_tag
-    suffix = random_string(4)
-    benchmark = get_benchmark(args, benchmark_definitions)
-    master_random_seed = get_master_random_seed(args.random_seed)
+    if extra_args is None:
```

Comment:
This seems the same code as in `hpo_main_...`. Could be a function?

Reply:
Yes, changed as part of list/None fix